students/sanghyun/users/views.py

```diff
@@ -1,3 +1,32 @@
-from django.shortcuts import render
+import json
+import re
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from users.models import User
+
+class SignUpView(View):
+    def post(self, request):
+        data           = json.loads(request.body)
+        check_email    = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
+        check_password = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
```

Comment:
아래와 같이 상수는 대문자로 선언해주시고, 위치는 SignupView class 밖으로 빼주세요-!
그 이유는 다른 곳에서 사용할 수 있기 때문입니다~
```suggestion
        EMAIL_REGEX    = '^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$'
        PASSWORD_REGEX = '^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$'
```

Reply:
대문자 수정 및 위치 변경 했습니다.